.eslintrc.js

```diff
@@ -12,6 +12,7 @@ module.exports = {
   },
   rules: {
     // Place to specify ESLint rules. Can be used to overwrite rules specified from the extended configs
+    '@typescript-eslint/triple-slash-reference': 'off',
```

Comment:
:question: can we add a short comment to explain why we need this? I hope it's temporary

Reply:
may be not needed, removed from this PR anyway